core/frontend/src/main/scala/io/udash/bindings/inputs/RadioButtons.scala

```diff
@@ -0,0 +1,104 @@
+package io.udash.bindings.inputs
+
+import java.{util => ju}
+
+import io.udash._
+import io.udash.properties.PropertyCreator
+import org.scalajs.dom.{Element, Event, Node}
+import org.scalajs.dom.html.{Div, Input => JSInput}
+import scalatags.JsDom
+import scalatags.JsDom.all._
+
+/**
+  * Radio buttons group for finite options with one element selection.
+  */
+object RadioButtons {
+  def inputsOnlyDecorator[T]: Seq[(JSInput, T)] => Seq[Node] = _.map(_._1)
+  def spanWithLabelDecorator[T](labelFactory: T => Modifier): Seq[(JSInput, T)] => Seq[Node] =
+    _.map { case (in, v) => span(in, label(labelFactory(v))).render }
+  def divWithLabelDecorator[T](labelFactory: T => Modifier): Seq[(JSInput, T)] => Seq[Node] =
+    _.map { case (in, v) => div(in, label(labelFactory(v))).render }
+
+  /**
+    * @param selectedItem Property which gonna be bound to radio buttons group.
+    * @param options Seq of available options, one radio button will be created for each option.
+    * @param decorator Function creating HTML element from buttons Seq.
+    *                  (Check: `RadioButtons.inputsOnlyDecorator`, `RadioButtons.spanWithLabelDecorator` and `RadioButtons.divWithLabelDecorator`)
+    * @param xs Modifiers to apply on each generated checkbox.
+    * @return HTML element created by decorator.
+    */
+  def apply[T : PropertyCreator](
+    selectedItem: Property[T], options: ReadableProperty[Seq[T]]
+  )(decorator: Seq[(JSInput, T)] => Seq[Node], xs: Modifier*): InputBinding[Div] = {
+    new InputBinding[Div] {
+      private val buttons = div(
+        nestedInterceptor(
+          produceWithNested(options) { case (opts, nested) =>
+            if (opts.nonEmpty && !opts.contains(selectedItem.get)) {
+              selectedItem.set(opts.head)
+            }
+
+            decorator(
+              opts.zipWithIndex.map { case (opt, idx) =>
+                val in = input(
+                  xs, tpe := "radio", value := idx.toString,
+                  nested((checked := "checked").attrIf(selectedItem.transform(_ == opt)))
+                ).render
+
+                in.onchange = (_: Event) => selectedItem.set(opt)
+
+                (in, opt)
+              }
+            )
+          }
+        )
+      ).render
+
+      override def render: Div = buttons
+    }
+  }
```

Comment:
This is almost 1:1 the code from `CheckButtons`. Maybe we could introduce an abstraction?